shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.test.js

```diff
@@ -232,4 +236,54 @@ describe('fileCourtIssuedDocketEntryInteractor', () => {
       caseTitle: Case.getCaseTitle(caseRecord.caseCaption),
     });
   });
+
+  it('should add docketEntry to caseEntity when not already on caseEntity', async () => {
+    const LEAD_CASE = {
+      ...MOCK_LEAD_CASE_WITH_PAPER_SERVICE,
+      docketEntries: [
+        {
+          docketEntryId: 'b01afa63-931e-4999-99f0-c892c51292d6',
+          docketNumber: '109-19',
+          documentTitle: 'Some Title',
+          documentType: 'Trial Exhibits',
+          eventCode: 'TE',
+          signedAt: '2019-03-01T21:40:46.415Z',
+          signedByUserId: mockUserId,
+          signedJudgeName: 'Dredd',
+          userId: mockUserId,
+        },
+      ],
+    };
+
+    applicationContext
+      .getPersistenceGateway()
+      .getCaseByDocketNumber.mockResolvedValueOnce(LEAD_CASE);
+
+    applicationContext
+      .getPersistenceGateway()
+      .getCaseByDocketNumber.mockResolvedValueOnce(
+        MOCK_CONSOLIDATED_1_CASE_WITH_PAPER_SERVICE,
+      );
+
+    await fileCourtIssuedDocketEntryInteractor(applicationContext, {
+      documentMeta: {
+        docketEntryId: LEAD_CASE.docketEntries[0].docketEntryId,
+        docketNumbers: [
+          MOCK_CONSOLIDATED_1_CASE_WITH_PAPER_SERVICE.docketNumber,
+          LEAD_CASE.docketNumber,
+        ],
+        documentType: 'Trial Exhibits',
+        eventCode: 'TE',
+        subjectDocketNumber: '109-19',
+      },
+    });
+
+    expect(
+      applicationContext
+        .getUseCaseHelpers()
+        .updateCaseAndAssociations.mock.calls[0][0].caseToUpdate.docketEntries.find(
+          docketEntry => docketEntry.eventCode === 'TE',
+        ),
+    ).toBeDefined();
```

Comment:
We should expect other things like ensuring that the `docketNumber` was updated for the different case.